src/kernels/backproject.cl

```diff
@@ -64,9 +64,7 @@ backproject_tex (read_only image2d_t sinogram,
     const float by = idy - axis_pos + y_offset + 0.5f;
     float sum = 0.0f;
 
-#ifdef DEVICE_GEFORCE_GTX_TITAN_BLACK
-#pragma unroll 8
-#endif
+#pragma unroll 14
```

Comment:
On a normal one. I don't want to use `#else` because unrolling by 14 is for sure not the best option for *every* other card than the Titan Black. I also tried `DEVICE_GEFORCE_GTX_TITAN` but it doesn't work. Do you know where are these definitions for the compiler?